packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -218,7 +233,16 @@ export default class SalesforceCRMService implements CRM {
           WhoId: firstContact,
         }).catch((reason) => Promise.reject(reason));
       }
-      return Promise.reject(reason);
+      log.error(`Error creating event: ${JSON.stringify(reason)}`);
+
+      // Try creating a simple object without additional records
+      await this.salesforceCreateEventApiCall(event, {
```

Comment:
The `System.NullPointerException: Attempt to de-reference a null object` log only comes from events that are writing to the event object. If we're seeing this error let's try and create a simple event and determine what is going wrong